src/ensembl/src/content/app/entity-viewer/components/entity-viewer-topbar/EntityViewerTopbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ScaleSwitcher from './scale-switcher/ScaleSwitcher';
+import FeatureSummaryStrip from 'src/shared/components/feature-summary-strip/FeatureSummaryStrip';
+
+import { RootState } from 'src/store';
+import { EnsObject } from 'src/shared/state/ens-object/ensObjectTypes';
+import { getEntityViewerActiveEnsObject } from '../../state/general/entityViewerGeneralSelectors';
```

Comment:
When you find yourself going several directories up, it's better import modules by starting from `src`.